index.html

```diff
@@ -210,12 +210,16 @@ <h1>DRAFT</h1>
         <section id="scope" class="scope">
             <h2>Scope</h2>
 
+            <p>
+                A terminological note: in what follows in this charter, and in the terminology to be used by the Working Group, the term “Linked Data” is used as a synonym to “RDF”.   
```

Comment:
Suggestion: the terms “Linked Data” and “RDF” are both used to refer to connected data on the web.

(weaker than "synonym" which is sometimes used in more formal contexts)